db/migrate/20161027134135_add_specialist_publisher_change_notes.rb

```diff
@@ -0,0 +1,7 @@
+class AddSpecialistPublisherChangeNotes < ActiveRecord::Migration[5.0]
+  def change
```

Comment:
Why `change`, and not say `up` as I would guess that you cannot roll this back? (I did try locally, but nothing seemed to happen, which I guess could be fine?)